docs/root/start/sandboxes/csrf.rst

```diff
@@ -34,47 +34,88 @@ The following documentation runs through the setup of both services.
 
 **Step 1: Install Docker**
 
-Ensure that you have a recent versions of ``docker`` and ``docker-compose``.
+Ensure that you have a recent versions of ``docker``, ``docker-compose``, and ``docker-machine``.
 
 A simple way to achieve this is via the `Docker Toolbox <https://www.docker.com/products/docker-toolbox>`_.
 
-**Step 2: Clone the Envoy repo and start all of our containers**
+**Step 2: Setup Docker Machines**
```

Comment:
We eliminated references to Docker Machine in https://github.com/envoyproxy/envoy/commit/173765d33e9ae1f437d3a55c0ee79b3ca19a3eb8. It is a very heavy weight way to test these sandboxes and requires an additional virtualization layer. Can both of these of terminals just use Docker and different ports? This is what we moved to for the CORS sandbox at https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/cors